Tests/SeedeeKitTests/ExportArchiveXcodeProjectActionTests.swift

```diff
@@ -4,7 +4,51 @@ import AppStoreConnect_Swift_SDK
 @testable import SeedeeKit
 
 final class ExportArchiveXcodeProjectActionTests: XCTestCase {
-    private let configuration = APIConfiguration.Key(privateKeyID: UUID().uuidString, issuerID: UUID().uuidString, privateKey: "MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgPaXyFvZfNydDEjxgjUCUxyGjXcQxiulEdGxoVbasV3GgCgYIKoZIzj0DAQehRANCAASflx/DU3TUWAoLmqE6hZL9A7i0DWpXtmIDCDiITRznC6K4/WjdIcuMcixy+m6O0IrffxJOablIX2VM8sHRscdr", path: nil)
+//    private let configuration = APIConfiguration.Key(privateKeyID: UUID().uuidString, issuerID: UUID().uuidString, privateKey: "MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgPaXyFvZfNydDEjxgjUCUxyGjXcQxiulEdGxoVbasV3GgCgYIKoZIzj0DAQehRANCAASflx/DU3TUWAoLmqE6hZL9A7i0DWpXtmIDCDiITRznC6K4/WjdIcuMcixy+m6O0IrffxJOablIX2VM8sHRscdr", path: nil)
```

Comment:
Test later because uploading to App Store Connect with `-exportArchive` with authentication key parameters isn't working.
https://developer.apple.com/forums/thread/690676
https://developer.apple.com/forums/thread/107976
https://stackoverflow.com/questions/70657042/ipa-upload-to-testflight-fails-via-jenkins-but-passes-via-terminal